python/kserve/requirements.txt

```diff
@@ -23,5 +23,6 @@ protobuf>=3.19.0
 prometheus-client>=0.13.1
 orjson>=3.8.0
 httpx>=0.23.0
-fastapi>=0.85.0,<=0.88.0
+fastapi>=0.89.1
+uvicorn>=0.16.0
```

Comment:
We can remove the uvicorn requirement, it is already specified by fastapi

Reply:
removed!